samples/caf_sensor_manager/app.overlay

```diff
@@ -5,6 +5,13 @@
  */
 
  / {
+	sensor_stub: sensor_stub {
+		compatible = "nordic,sensor-stub";
+		label = "SENSOR_STUB";
+		generator = "sensor_stub_gen";
+		status = "okay";
```

Comment:
Should both simulators have status okay? Wouldn't it be confusing for a user?

Reply:
In my opinion they should. The fact that sensor_sim generates its driver even if there is not "okay" status seems like an error. It is possible only because sensor_sim is single instance driver. But I have already removed that from the sensor_sim as this is another change that is not needed in my implementation.

Reply:
Discussed offline. Agreed this is currently best solution.